src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-660.mdx

```diff
@@ -0,0 +1,23 @@
+---
+subject: Android agent
+releaseDate: '2022-06-09'
```

Comment:
This PR looks fine! My only question is, the releaseDate says June 9. Do you want us to wait until the 9th to merge this PR, or would you like to change the date and release today? Thanks for confirming either way!

Reply:
Oh sorry, we can make it June 2. That was probably just the projected release date.